app/views/layout.erb

```diff
@@ -14,15 +14,21 @@
   <%# finally, make the page-top bar somewhat respectably toned down too %>
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
 
-  <script src="jquery.js"></script>
-  <script src="jquery.cookie.js"></script>
-  <script src="pages.js"></script>
+  <!-- <script src="jquery.js"></script> -->
+  <script src="//code.jquery.com/jquery-1.11.2.js"></script>
```

Comment:
:+1: from loading from CDN. You may remove the old vendored `jquery.js` and references to it.